ui/app/index.html

```diff
@@ -19,12 +19,16 @@
             var app = Elm.Main.init({
                 flags: {
                     production: true,
-                    defaultCreator: localStorage.getItem('defaultCreator')
+                    defaultCreator: localStorage.getItem('defaultCreator'),
+                    groupExpandAll: JSON.parse(localStorage.getItem('groupExpandAll'))
```

Comment:
After a lot of anger and confusion, I did some reading and figured out that booleans cannot be stored into local storage. Is this an ok way to get the boolean back out? Not sure if there's a convention around it.